files/en-us/mdn/structures/macros/commonly-used_macros/index.html

```diff
@@ -48,21 +48,86 @@ <h3 id="Creating_a_single_hyperlink">Creating a single hyperlink</h3>
 
 <h3 id="Linking_to_pages_in_references">Linking to pages in references</h3>
 
-<p>There are various macros for linking to pages in specific reference areas of MDN.</p>
+<p>There are macros for locale-independent linking to pages in specific reference areas of MDN: Javascript, CSS, HTML elements, SVG etc.</p>
+
+<p>The macros are easy to use. Minimally all you need to do is specify the name of the item to link to in the first argument. Most macros will also take a second argument allowing you to change the display text (documentation can be found at the links in the left-most column below).</p>
+
+<table class="standard-table">
+    <caption>Compound assignment operators</caption>
```

Comment:
I think captions are going away in markdown-land but for now we maybe want them still?

```suggestion
    <caption>MDN reference link macros</caption>
```

Reply:
Thanks - that's a copy paste error from when I copied the page. I will delete.